crates/red_knot_python_semantic/src/types/call/bind.rs

```diff
@@ -138,7 +138,7 @@ pub(crate) fn bind_call<'db>(
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub(crate) struct CallBinding<'db> {
     /// Type of the callable object (function, class...)
```

Comment:
I wonder if this will be true. E.g. it's already a function in the `Class.__call__` case. So maybe it's always the function type?

Reply:
Not sure I fully understand the question, but it's possible for the `__call__` method to be an instance of another callable object. It doesn't have to be a function (https://bsky.app/profile/alexwaygood.bsky.social/post/3lejkfqwhkk2o)